domains/digifriend.json

```diff
@@ -6,7 +6,7 @@
         },
     
         "record": {
-            "A": ["172.28.248.222"]
+            "A": ["208.67.222.222"]
```

Comment:
I cannot reach your website
> The request was directed to a path other than the configured endpoint

Reply:
are you sure this is your IP? @StarSupreme

Reply:
You know what, this is very confusing. I'll just delete this domain and get a new one.

Reply:
I mean, how would my website be accessible when I'm testing?! I'm trying to deploy my tensorflow-based flask app, so that I could see it in a website and submit it in a competition, but my wsl ip address doesn't work for some reason. I'll just delete this domain, and create a new one with the correct ip. Is that fine with you?